queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/__init__.py

```diff
@@ -48,27 +48,28 @@ def create_address(address_info: Dict, address_type: str) -> Address:
 
     db_address_type = address_type.replace('Address', '')
 
-    address = Address(street=address_info.get('streetAddress'),
-                      street_additional=address_info.get('streetAddressAdditional'),
-                      city=address_info.get('addressCity'),
-                      region=address_info.get('addressRegion'),
-                      country=pycountry.countries.search_fuzzy(address_info.get('addressCountry'))[0].alpha_2,
-                      postal_code=address_info.get('postalCode'),
-                      delivery_instructions=address_info.get('deliveryInstructions'),
-                      address_type=db_address_type
-                      )
+    address = Address(
+        street=address_info.get('streetAddress') or '',
```

Comment:
setting props to empty string if `None` to avoid creation of new version in the version table if the current version has empty string or vice versa

Reply:
@argush3 this need to be addressed in data migration as well.